packages/e2e/cypress/e2e/dashboard.cy.ts

```diff
@@ -69,6 +69,21 @@ describe('Dashboard', () => {
         cy.findByRole('dialog').get('.mantine-MultiSelect-input').click(); // Close dropdown
         cy.findByText('Add').click();
 
+        cy.findAllByText('Add tile').click({ multiple: true });
```

Comment:
should we make this a different test ? If it fails, I don't know if it is because the "chat in dashboard" feature is wrong, or is adding an existing title wrong.